roles/logging/handlers/main.yml

```diff
@@ -14,3 +14,8 @@
 
 - name: restart rsyslog
   service: name=rsyslog state=restarted
+
+- name: restart filebeat
+  service: name=filebeat state=restarted must_exist=false
+  failed_when: false
```

Comment:
agree, we should work service library to ensure service failures are not silently ignored under separate card.